pom.xml

```diff
@@ -136,6 +136,12 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<!-- TODO: Why isn't the version taken from boot bom? -->
+			<dependency>
+				<groupId>org.springframework</groupId>
+				<artifactId>spring-core</artifactId>
+				<version>4.3.4.RELEASE</version>
```

Comment:
That can happen if you use the wrong version of Maven. Are you sure you are using the wrapper, and does it have 3.3.9?